service/storage_service.cc

```diff
@@ -5930,10 +5930,11 @@ future<> storage_service::do_tablet_operation(locator::global_tablet_id tablet,
         _tablet_ops.erase(tablet);
     });
 
+    rtlogger.info("{} for tablet migration of {} starting", op_name, tablet);
     try {
         co_await op(guard);
         p.set_value();
-        rtlogger.debug("{} for tablet migration of {} successful", op_name, tablet);
+        rtlogger.info("{} for tablet migration of {} successful", op_name, tablet);
     } catch (...) {
```

Comment:
I think you possibly forgot to undo these changes before pushing.

Reply:
not really.
Ideally, I should have them in a separate patch and explain why.
This information is rare enough not to spam the log, and important enough to be logged at info rather than debug level, to provide the visibility to figure out what's going on in the system around tablet migrations.

Reply:
A possible spam scenario is high smp count and decommissions. storage_service::stream_tablet() which invokes do_tablet_op() happens to run on destination.